cargo/lib/dependabot/cargo/metadata_finder.rb

```diff
@@ -67,7 +67,12 @@ def crates_listing
           **SharedHelpers.excon_defaults(headers: hdrs)
         )
 
-        @crates_listing = JSON.parse(response.body)
```

Comment:
Output after this change
```bash
Calling https://cargo.cloudsmith.io/honeyankit/test/he/ll/hello-world to fetch metadata for hello-world from sparse+https://cargo.cloudsmith.io/honeyankit/test/
Fetched metadata for hello-world from sparse+https://cargo.cloudsmith.io/honeyankit/test/ successfully
{"name": "hello-world", "vers": "1.0.0", "deps": [], "cksum": "b2c263921f1114820f4acc6b542d72bbc859ce7023c5b235346b157074dcccc7", "features": {}, "yanked": false, "links": null}
{"name": "hello-world", "vers": "1.0.1", "deps": [], "cksum": "8a55b58def1ecc7aa8590c7078f379ec9a85328363ffb81d4354314b132b95c4", "features": {}, "yanked": false, "links": null}
{"name": "hello-world", "vers": "1.0.2", "deps": [], "cksum": "9e21213589c10d30973283629bb27520ddf46141882e26ea10b70959c7bafc2c", "features": {}, "yanked": false, "links": null}
2024/05/20 22:53:57 INFO Latest version is 1.0.2
Setting CARGO_REGISTRIES_HONEYANKIT_TEST_TOKEN to 'placeholder_token' because dependabot-cli proxy will override it anyway
2024/05/20 22:53:58 INFO Requirements to unlock own
2024/05/20 22:53:58 INFO Requirements update strategy bump_versions
2024/05/20 22:53:58 INFO Updating hello-world from 1.0.1 to 1.0.2
Setting CARGO_REGISTRIES_HONEYANKIT_TEST_TOKEN to 'placeholder_token' because dependabot-cli proxy will override it anyway
2024/05/20 22:53:58 INFO Submitting hello-world pull request for creation
2024/05/20 22:53:59 INFO Finished job processing
2024/05/20 22:53:59 INFO Results:
+-----------------------------------------------+
|      Changes to Dependabot Pull Requests      |
+---------+-------------------------------------+
| created | hello-world ( from 1.0.1 to 1.0.2 ) |
+---------+-------------------------------------+
```
```